### ISSUE TITLE
[1.16, Bug] Crash with Futurepack mod

### ISSUE THREAD

**Issue by sirben99** [GiantLuigi4/Smaller-Units#204]
-- Describe how you caused the bug --
When you have Futurepack mod (https://www.curseforge.com/minecraft/mc-mods/futurepack) and Smaller-Units mod installed, putting an block into a Smaller-Units tile causes the game to crash.

The Unit tiles are also broken with Futurepack installed. In the screenshot i have 1/3 unit on the left, and 1/16 unit on the right, but theyr both the same size. This does not happen when only Smaller-Units is installed.
![2023-07-27_13 21 33](https://github.com/GiantLuigi4/Smaller-Units/assets/32977052/719e91ea-cca3-425b-aea1-c43c18914731)

If this is something that should be reported to Futurepack instead, please let me know and i will re-direct this!

-- Describe the expectation --
The game doesent crash when placing blocks on unit tiles and the unit tiles are the intended size

-- Describe what happened instead --
The game crashes when placing blocks on unit tiles and the unit tiles are not the intended size

-- Additional info --
SU Version: 1.3
Futurepack Version: 31.3.7008
Exact MC Version: 1.16.5
Loader: Forge
Any optimization mods: No

-- Logs if the game crashed --
Crash report: https://pastebin.com/PetJAX89
Log: https://pastebin.com/1Lcejaq6

Thank you for your time!

**Comment by GiantLuigi4** [comment]
I… am pretty sure I fixed that crash at some point, but I guess I never uploaded it?
It’s more so a forge version thing than a modpack thing.
As for them both being 1/4, that is odd, I have not seen that

**Comment by sirben99** [comment]
Yeah, its a super weird issue. Two things i forgot to mention in my initial post: 

1. I dont have other mods installed than these two
2. Im running the latest 1.16.5 forge, which is 36.2.39

If you need any more info on the subject please ask!

**Comment by GiantLuigi4** [comment]
SU was truthfully designed for forge 36.2.20
I believe forge update since then and made a change which happened to make, probably truthfully a large standing issue, crash the game the moment anyone interacts with the unit spaces

**Comment by sirben99** [comment]
That could explain it. The crash only happens with Futurepack installed, Small-Units works fine standalone

**Comment by GiantLuigi4** [comment]
Ok nvm then
That makes this… a lot more interesting
I’ll have to look at it at some point, but at this point I’m mostly fixing simple issues with the 1.16 version, so I don’t have any guarantee

**Comment by sirben99** [comment]
Np! Take your time with the fixes, better safe than sorry. Thank you for looking into it!
